cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectDockerfile.java

```diff
@@ -264,9 +259,9 @@ private void addOtherArtifact(PrintWriter writer, String connectorPath, OtherArt
         String artifactDir = connectorPath + "/" + artifactHash;
         String fileName = other.getFileName() != null ? other.getFileName() : artifactHash;
         String artifactPath = artifactDir + "/" + fileName;
-        String downloadCmd =  "curl -L --output " + artifactPath + " " + other.getUrl();
+        Cmd downloadCmd =  new Cmd("curl", "-L", "--output").append(artifactPath).append(other.getUrl());
```

Comment:
We don't really need the `.append` here, they're just extra arguments to the `...` parameter.

Reply:
Also, doesn't `artifactPath` depend on untrusted input?